src/components/Footer/Footer.jsx

```diff
@@ -1,8 +1,52 @@
 import React from 'react';
 import styles from './Footer.module.css';
+import { Link } from 'react-router-dom';
 
 const Footer = () => {
-	return <div className={styles.footer}>Footer - primary navigation</div>;
+	return (
+		<div className={styles.footer}>
+			<Link to='/'>
+				<div className={styles.link}>
+					<img
+						className={styles.icon}
+						src='../../src/assets/icons/home-circle.svg'
+						alt='Home inside a circle'
+					/>
+					<span>Home</span>
+				</div>
+			</Link>
+			<Link to='/alltrips'>
```

Comment:
`<Link className={pathname === '/alltrips' ? styles.active : null} to='/mytrips'>`